src/hooks/useId.ts

```diff
@@ -1,9 +1,16 @@
-import { useRef } from "react";
-import { nanoid } from "nanoid";
+import { useId as useIdReact } from "react";
+
+import { IS_DEV } from "utils";
 
 /**
- * A hook that returns the same random ID string on every render.
- * Can be used as a value for HTML "id" attributes.
- * @returns random ID string
+ * @deprecated Code component is deprecated. Use CodeSnippet component or inline `<code>` instead.
  */
-export const useId = (): string => useRef(nanoid()).current;
+export const useId = () => {
```

Comment:
After a quick look at the imports, I'm not sure if this is used anywhere right now. Is there a reason why we still need this code?

Reply:
This is just for any external projects that use this hook (I know some projects like maas-ui and juju-dashboard used to import it, though I'm not sure if they still do).